frontend/assets/style/components/_forms.scss

```diff
@@ -300,29 +336,76 @@ p.error {
       margin-right: $spacer-sm;
       border: 1px solid $general_0;
       transform: translateY(2px);
+      transition: box-shadow 150ms, background-color 150ms;
     }
 
     &::after {
       position: absolute;
       width: 6px;
       height: 6px;
-      background-color: $primary_0;
+      background-color: $background_0;
       top: 11px;
       left: 7px;
       transform: translate(-50%, -50%) scale(0);
-      transform-origin: 50% 50%;
       transition: transform 150ms;
     }
   }
 
-  .input.is-checked + :last-child,
-  input:focus + :last-child,
-  input:checked + :last-child {
-    font-weight: bold;
-    margin-right: -1px;
+  .input:checked + :last-child {
+    &::before {
+      border-color: $primary_0;
+      background-color: $primary_0;
+    }
+
+    &::after {
+      transform: translate(-50%, -50%) scale(1);
+    }
+  }
+}
+
+.checkbox {
+  :last-child {
+    white-space: nowrap;
+
+    &::before,
+    &::after {
+      display: inline-block;
+      border-radius: 2px;
+    }
 
+    &::before {
+      content: "";
+      width: 14px;
+      height: 14px;
+      margin-right: $spacer-sm;
+      border: 1px solid $general_0;
+      transform: translateY(2px);
+      transition: box-shadow 150ms, background-color 150ms;
+    }
+
+    &::after {
+      content: "\f00c"; // icon-check
+      text-rendering: auto;
+      line-height: 1;
+      font-family: "Font Awesome 5 Free";
```

Comment:
should we use icon-check instead?

Reply:
What do you mean? We use it on line 387 `content: "\f00c"; // icon-check`.

Or do you mean to add the element in the DOM? `<i class="icon-check">` I think it'd be too much verbose because each time we need to use .checkbox, we would need to conditionally hide that element.

Reply:
I was talking about the check. But I think you're right. Maybe a mixin could be a good compromise, but no need to do that in the PR I guess

Reply:
yah a mixin sounds better, AFAIK, we have 3-4 places in our codebase using the icons this way. I can address all those places in a different PR